fulp_modules/hardhat_nolight/lampless_hardhat.dm

```diff
@@ -0,0 +1,11 @@
+/obj/item/clothing/head/lampless_hardhat
```

Comment:
why isnt this a hardhat subtype?

Reply:
![image](https://user-images.githubusercontent.com/68878861/107786577-5b34dd80-6d4e-11eb-8780-3f21c64a7593.png)

Reply:
just override those, if you do it like this some stuff like tackles and psychotic brawls will not take this helmet in consideration

Reply:
We've talked this over now and they still are against it.

Reply:
that's silly

Reply:
lol

Reply:
> 
> 
> We've talked this over now and they still are against it.

just override the functionality you don't want or restructure so lampless is the parent type

Reply:
Even when I asked how would I go on about this, Pucci and Pengu discourage it, Willard claims it might even make the code messier. I won't resolve this conversation yet, but for now it's probably a no-no.

Reply:
they are wrong

Reply:
realistically there should be a hardhat subtype and a hardhat/lamp subtype.